src/index.html

```diff
@@ -1,13 +1,57 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0,   maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+
+  <body class="content">
+    <div class="card-content" data-qa="card">
+      <img 
+      class="card-content__image"
+      src="images/imac.jpeg"
+      alt="iMac"
+      >
+      <div class="card-content__produkt">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="card-content__code">
+        Product code: 195434
+      </div>
+      <div class="card-content__reviews">
+        <div  class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="number">
+        Reviews: 5
+        </div>
+      </div>
+      <div class="card-content__price">
+        <div class="card-content__price-name">
+          Price:
+        </div>
+        <div class="card-content__price-cost">
+          $2,199
+        </div>
+      </div>
```

Comment:
Checklist p.3
also use semantic tags (checklist p.12)
Please apply everywhere too
```suggestion
      <div class="card-content__price">
        <p class="card-content__price-name">
          Price:
        </p>
        
        <p class="card-content__price-cost">
          $2,199
        </p>
      </div>
```
![image](https://user-images.githubusercontent.com/118751034/220118730-7e9e8acd-072b-46b9-80c8-b0fe316777cb.png)

Reply:
previous comment wasn't fixed